Library/Formula/kibana.rb

```diff
@@ -0,0 +1,63 @@
+class Kibana < Formula
+  desc "Visualization tool for elasticsearch"
+  homepage "https://www.elastic.co/products/kibana"
+  url "https://github.com/elastic/kibana/archive/v4.0.2.tar.gz"
+  sha256 "facb5b2eb31de38ebf6e30bb656c62a512e540623df98b33655f5b0da896dd27"
+  head "https://github.com/elastic/kibana.git"
+
+  depends_on "node"
+
+  def install
+    ENV.prepend_path "PATH", HOMEBREW_PREFIX/"bin"
+    system "npm", "install"
+    system "npm", "install", "grunt-cli"
+    system "npm", "install", "bower"
+    system "./node_modules/.bin/bower", "install"
+    system "./node_modules/.bin/grunt", "build", "--force"
+
+    dist_dir = Pathname("build/dist/kibana")
+
+    rm_f dist_dir/"bin/*.bat"
+
+    prefix.install dist_dir/"src"
+    (etc/"kibana").mkpath
+    (var/"lib/kibana/plugins").mkpath
+
+    (etc/"kibana").install dist_dir/"config/kibana.yml" unless (etc/"kibana/kibana.yml").exist?
+
+    # point to our node
+    inreplace dist_dir/"bin/kibana" do |s|
+      s.sub! /^NODE=.*$/, "NODE=#{Formula["node"].opt_bin}/node"
+    end
+
+    bin.install dist_dir/"bin/kibana"
+  end
+
+  def post_install
```

Comment:
If they're in `var` and `etc` they won't get removed across upgrades. Only the prefix is wiped in that situation, or in a `brew rm kibana` situaiton. Do they need to be symlinked back to the prefix for the binary to correctly find them?